external/plex/dist/Plex/test_tm.py

```diff
@@ -1,7 +1,7 @@
 import sys
 sys.stderr = sys.stdout
 
-from TransitionMaps import TransitionMap
+from Transitions import TransitionMap
```

Comment:
Date: Fri, 02 Oct 2015 05:11:55 -0700
   From: axch <notifications@github.com>

   So, I'm instinctively against modifying third-party code, because I
   assume merging those modifications with upstream would be a
   nightmare.  @riastradh-probcomp , do you expect that with the way
   you set this up, my assumption is just wrong?

Yes -- that's why the procedure in external/README is the way it is
(although it's easier if we restrict changes to be within the
external/ directory).  To find all our local changes to foo:

$ git log --patch vendor/foo..master -- ./external/foo/dist

   Also, should I assume that we will merge upstream patches rarely
   enough that my fear is irrelevant even if it is somewhat justified?

In the case of plex, I don't know.  Lemonade doesn't seem to be
maintained any more, although the author did respond at first when I
contacted him about it and once upon a time I intended to feed my
changes upstream.